rest-twitter/post_tweets_local_db.py

```diff
@@ -0,0 +1,38 @@
+from pymongo import MongoClient
+from datetime import datetime
+
+def get_timestamp():
+    return datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
+
+client = MongoClient()
+
+db = client.tweet_store
+
+tweets = db.tweets
+
+dummy_tweet_1 = {
+        "woeid": "2367231",
```

Comment:
The attribute names should be Capital case! Good practice while designing RESTful services